app/views/contexts/thank_you_context.py

```diff
@@ -19,23 +18,28 @@
 
 def build_thank_you_context(
     schema: QuestionnaireSchema,
-    session_data: SessionData,
+    questionnaire_store_metadata: dict,
```

Comment:
Just call it `metadata` like we do elsewhere?

Reply:
Renamed it but there is another metadata var later in the method which I also had to rename